src/pages/_app.tsx

```diff
@@ -41,12 +48,15 @@ export default function Intellect({ Component, pageProps }: AppPropsWithLayout)
                     name="keywords"
                     content="система Інтелект, НТУУ КПІ, Київський політехнічний інститут, викладачі КПІ, Intellect"
                 />
-                <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
+                <meta name="viewport"
+                    content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
```

Comment:
why this needed ?

Reply:
Seems this line was accidentally moved from one line to another.

In general, this prevents jumping scaling while focusing on different user controls.